controller/backing_image_manager_controller.go

```diff
@@ -820,6 +824,16 @@ func (c *BackingImageManagerController) generateBackingImageManagerPodManifest(b
 							MountPath: bimtypes.DiskPathInContainer,
 						},
 					},
+					Env: []v1.EnvVar{
+						{
+							Name: types.EnvPodIP,
+							ValueFrom: &v1.EnvVarSource{
+								FieldRef: &v1.ObjectFieldSelector{
+									FieldPath: "status.podIP",
```

Comment:
Where is the env var used? If for data sync, why not storageIP?

Reply:
We should allow the BIM to fall back to using the pod IP when the Storage Network is not launched.
https://github.com/shuo-wu/backing-image-manager/blob/6dc15c6ec60a126590ac117214122c1c1a15e5cd/pkg/util/http_util.go#L59

Reply:
cc @c3y1huang